pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -17,9 +17,13 @@
 let
   py = python3.override {
     packageOverrides = self: super: {
-      django = super.django_3;
+      django = super.django_4;
+
+      # django-extensions 3.1.5 is required by paperless-ngx, but its tests are incompatible with Django 4
+      django-extensions = super.django-extensions.overridePythonAttrs (_: {
+        doCheck = false;
+      });
 
-      # Incompatible with aioredis 2
```

Comment:
actually, paperless-ngx is still incompatible with aioredis 2. Maybe we should leave the comment

Reply:
I had removed it because:
- It is missing a subject and thus sounds confusing to me, even though I'm familiar with the code.
- The Django package override would require a similar comment (`paperless-ngx is incompatible with Django 3`).

I've now added a similar comment to the whole package override section.